services/hierarchy/src/main/kotlin/OrtRunService.kt

```diff
@@ -73,4 +77,22 @@ class OrtRunService(
             throw ResourceNotFoundException("ORT run with id '$ortRunId' not found.")
         }
     }
+
+    /**
+     * Delete all ORT runs that are older than the given [before] timestamp. Runs are deleted from the database and
+     * their reports are deleted from storage.
+     */
+    suspend fun deleteRunsCreatedBefore(before: Instant) {
+        val operator = FilterOperatorAndValue(ComparisonOperator.LESS_THAN, before)
+        val listQueryParameters = ListQueryParameters.DEFAULT.copy(limit = Int.MAX_VALUE)
+        val runs = ortRunRepository.list(listQueryParameters, OrtRunFilters(finishedAt = operator))
+
+        logger.info("Deleting ${runs.totalCount} ORT runs older than $before.")
+
+        runs.data.forEach { run ->
+            runCatching {
+                deleteOrtRun(run.id)
+            }
```

Comment:
Exceptions when deleting a run should be logged.

Reply:
They are already logged in `deleteOrtRun`. I would like to avoid double logging.